pkgs/games/cataclysm-dda/git.nix

```diff
@@ -2,9 +2,9 @@
 , tiles ? true, Cocoa
 , debug ? false
 , useXdgDir ? false
-, version ? "2021-07-03"
-, rev ? "9017808252e1e149446c8f8bd7a6582ce0f95285"
-, sha256 ? "0qrvkbyg098jb9hv69sg5093b1vj8f4n75p73v01jnmyxlz3ax28"
+, version ? "2022-08-19"
+, rev ? "725e825d6f77d7d8cd70b9be39a5e67d5186969d"
+, sha256 ? "sha256-h+swzUuxP7xz0iYTD2mRNgxZp47fVJ1zHiqLWOlc9Z0="
```

Comment:
We probably had that discussion before but this should be inlined and people wishing to change it can use overrideAttr.

Reply:
I think it's done this way because using `overrideAttrs` breaks `passthru`, which for CDDA contains a wrapper for bundling mods, as well as a builders for mods themselves (there's [a section it the Nixpkgs manual](https://nixos.org/manual/nixpkgs/unstable/#important-note-for-overriding-packages)).

I guess we may be able to solve this issue with overlay-style overrideable attributes now, although that's a slightly broader project.